src/ngraph/ngraph_compiler.cc

```diff
@@ -386,6 +386,7 @@ void Compiler::DeepCopy(const nnvm::Graph& graph) {
 
 // Check nodes in NGraph
 void Compiler::CheckInNgraph() {
+  std::unordered_set<std::string> ops;
```

Comment:
It took me a little while to figure out what's going on here, and I think it was due to these factors:
* The name `ops` is pretty generic, even though its purpose is more along the lines of "unsupported ops in this graph."
* The code structure seems strange / overcomplicated to me, because it's not taking full advantage of the way `std::set` works.

I wonder if the following would be easier for most people to quickly apprehend?
``` c++
std::set<std::string> unsupported_ops_names;
...
for (auto node : ngraph_.nodes) {
   ...
  } else if (ngraph_log_verbose} {
    unsupported_op_names.insert(node->operation_);
  }
}
...
for (const auto & name : unsupported_op_names) {
   std::cout << "NGRAPH_BRIDGE: Unsupported Op: " << name << std::endl;
}
```